lib/sqlite3/sqlite3.nit

```diff
@@ -276,6 +280,8 @@ class StatementIterator
 			is_ok = false
 		end
 	end
+
+	redef fun finish do statement.close
```

Comment:
you should document somewhere that after one iteration, the whole statement is closed